lib/record_store/provider/dynect.rb

```diff
@@ -2,6 +2,10 @@
 
 module RecordStore
   class Provider::DynECT < Provider
+    def self.record_types
+      super.add('ALIAS')
```

Comment:
@es re: this comment ... I'm really not sure how we can tell that the zone supports ALIAS records (in part because it also requires explicit permission to manage them - not just that the zone has to "support" them)

did you have any idea(s) for this?